src/AppBundle/Entity/TaskList.php

```diff
@@ -81,4 +81,30 @@ public function removeTask(Task $task)
 
         return parent::removeTask($task);
     }
+
+    public function updateTasks($tasksToAssign)
```

Comment:
IMHO, having a `SplObjectStorage` as parameter is too specific. 
The `updateTasks` method should take a zero-indexed `array`, and use `SplObjectStorage` internally.

Reply:
Ok will modify